dojo/unittests/tools/test_aqua_parser.py

```diff
@@ -57,3 +57,63 @@ def test_aqua_parser_cvssv3_has_many_findings(self):
                     nb_cvssv3 = nb_cvssv3 + 1
 
             self.assertEqual(16, nb_cvssv3)
+
+    def test_aqua_parser_for_aqua_severity_critical(self):
+        with open("dojo/unittests/scans/aqua/vulns_with_aqua_severity.json") as testfile:
+            parser = AquaParser()
+            findings = parser.get_findings(testfile, Test())
+            nbsev = 0
+
+            for finding in findings:
+                if finding.severity == 'Critical':
+                    nbsev = nbsev + 1
+
+            self.assertEqual(1, nbsev)
+
+    def test_aqua_parser_for_aqua_severity_high(self):
+        with open("dojo/unittests/scans/aqua/vulns_with_aqua_severity.json") as testfile:
+            parser = AquaParser()
+            findings = parser.get_findings(testfile, Test())
+            nbsev = 0
+
+            for finding in findings:
+                if finding.severity == 'High':
+                    nbsev = nbsev + 1
+
+            self.assertEqual(1, nbsev)
+
+    def test_aqua_parser_for_aqua_severity_medium(self):
+        with open("dojo/unittests/scans/aqua/vulns_with_aqua_severity.json") as testfile:
+            parser = AquaParser()
+            findings = parser.get_findings(testfile, Test())
+            nbsev = 0
+
+            for finding in findings:
+                if finding.severity == 'Medium':
+                    nbsev = nbsev + 1
+
+            self.assertEqual(2, nbsev)
+
+    def test_aqua_parser_for_aqua_severity_low(self):
+        with open("dojo/unittests/scans/aqua/vulns_with_aqua_severity.json") as testfile:
+            parser = AquaParser()
+            findings = parser.get_findings(testfile, Test())
+            nbsev = 0
+
+            for finding in findings:
+                if finding.severity == 'Low':
+                    nbsev = nbsev + 1
+
+            self.assertEqual(2, nbsev)
+
+    def test_aqua_parser_for_aqua_severity_info(self):
+        with open("dojo/unittests/scans/aqua/vulns_with_aqua_severity.json") as testfile:
+            parser = AquaParser()
+            findings = parser.get_findings(testfile, Test())
+            nbsev = 0
+
+            for finding in findings:
+                if finding.severity == 'Info':
+                    nbsev = nbsev + 1
+
+            self.assertEqual(7, nbsev)
```

Comment:
Could you merge these into 1 test case `...for_severities` and use the `.count(...)` method of the findings list to assert the result?
i.e. option 3 from https://www.geeksforgeeks.org/python-count-occurrences-element-list/

Reply:
will do !